sphinx/_static/custom.css

```diff
@@ -0,0 +1,14 @@
+/* override table width restrictions */
```

Comment:
This prevents tables from being wider than their container, and thus prevents them requiring scrolling.